packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -424,11 +425,22 @@ export class DebugSessionManager {
             return this.doPostTaskAction(`Could not run the task '${taskName}'.`);
         }
 
-        const code = await this.taskService.getExitCode(taskInfo.taskId);
-        if (code === 0) {
+        const getExitCodePromise: Promise<TaskEndedInfo> = this.taskService.getExitCode(taskInfo.taskId).then(result =>
+            ({ taskEndedtype: TasksEndedTypes.TaskExited, value: result }));
```

Comment:
[minor] please rename to `taskEndedType` following the [guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#camelcase-var).

Reply:
fixed, done